lib/logstash/filters/translate.rb

```diff
@@ -56,9 +56,10 @@ class LogStash::Filters::Translate < LogStash::Filters::Base
   # NOTE: it is an error to specify both `dictionary` and `dictionary_path`
   config :dictionary, :validate => :hash,  :default => {}
 
-  # The full path of the external dictionary file. The format of the table
-  # should be a standard YAML, JSON or CSV. Make sure you specify any integer-based keys
-  # in quotes. For example, the YAML file should look something like this:
+  # The full path of the external dictionary file, the filename must end in `.yaml`
```

Comment:
...or `.yml`. Also, for JSON files it's `.json` and for CSV files `.csv`. Could you rephrase this sentence?